app/Http/Requests/CreateCommentRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+class CreateCommentRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
If you check here if the user is logged in, you wouldn't need the `Auth::guest()` check in `ProjectCommentsController` for creating the comment.